main/webapp/modules/core/scripts/index/parser-interfaces/xml-parser-ui.js

```diff
@@ -131,6 +131,7 @@ Refine.XmlParserUI.prototype._initialize = function() {
     this._optionContainerElmts.includeFileSourcesCheckbox.prop("checked", true);
   }
   this._optionContainerElmts.pickRecordElementsButton.click(function() {
+    self._config.recordPath = undefined;
```

Comment:
I did this because if I did `setRecordPath`, the flow reaches `updatePreview` first and then the same thing happened. Also I let the check in `confirmCreateProject` be beacuse that would be required when someone directly clicks that button.